src/explore-education-statistics-admin/src/pages/sign-in/SignInPage.tsx

```diff
@@ -13,12 +13,9 @@ const SignInPage = () => {
         Use this service to publish official Department for Education (DfE)
         statistics and data for state-funded schools in England.
       </p>
-      <ButtonLink
-        to="/tools/azuread/account/signin"
-        className="govuk-button--start"
-      >
+      <a href="/api/signin" className="govuk-button govuk-button--start">
```

Comment:
Just need  to remove the unused ButtonLink import

Reply:
Removed